go.mod

```diff
@@ -39,3 +39,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.6.4
 	sigs.k8s.io/controller-tools v0.4.1
 )
+
+require google.golang.org/grpc v1.27.1
```

Comment:
Any reason not to put this in the main require block with a comment explaining why it is needed?
Or is this just go.mod being sticky?

Reply:
Nope; I just hardcoded for my first pass. Already gone.